packages/roosterjs-editor-dom/lib/utils/tableInfo.ts

```diff
@@ -0,0 +1,42 @@
+import { TableFormat } from 'roosterjs-editor-types';
+
+const TABLE_STYLE_INFO = 'roosterTableInfo';
+
+/**
+ * Save the format info of a table
+ * @param table The table the info will be saved
+ * @param format The format of the table
+ */
+export function saveTableInfo(table: HTMLTableElement, format: Partial<TableFormat>) {

```

Comment:
If we can move the plugin to be regular, these code can be shared with ImageEditPlugin

Reply:
do you mean create a plugin to save data?